tests/test_traceback.py

```diff
@@ -307,6 +308,41 @@ def test_suppress():
         assert "foo" in traceback.suppress[1]
 
 
+@pytest.mark.parametrize(
+    "rich_traceback_omit_for_level2,expected_frames_length,expected_frame_names",
+    (
+        # fmt: off
+        [True, 3, ["test_rich_traceback_omit_optional_local_flag", "level1", "level3"]],
+        [False, 4, ["test_rich_traceback_omit_optional_local_flag", "level1", "level2", "level3"]],
+        # fmt: on
```

Comment:
I added these `# fmt: ` annotations just to avoid Black changing the second test case so that it spawns across multiple lines when the 1st one doesn't, as they are functionally the same thing
But happy to let Black format _everything_ and remove those if you prefer, of course :slightly_smiling_face:

Reply:
I like this. Black completely ruins the readability of some parametrised tests

Reply:
Fine by me!